src/main/java/io/gravitee/reporter/api/v4/metric/Metrics.java

```diff
@@ -75,6 +75,10 @@ public class Metrics extends AbstractReportable {
     private long requestContentLength = -1;
 
     private boolean requestEnded;
+    /**
+     * Entrypoint metrics
+     */
+    private String entrypointId;
```

Comment:
As we have `endpoint` on the other side, would it be more relevant to name it `entrypoint`?

Reply:
I was hesitating.
moreover, the name endpoint is a bit confusing here because it seems to be the resolved URL of the endpoint, and not the endpoint id

Reply:
You're right 👍